critiquebrainz/frontend/templates/review/modify/artist.html

```diff
@@ -1,11 +1,8 @@
-{% if entity is not defined %}
-  {% set entity = review.entity_id | entity_details(type=entity_type) %}
-{% endif %}
 <div class="col-md-12">
   <dl class="dl-horizontal">
     <dt>{{ _('Artist') }}</dt>
-    <dd>
-      {{ entity['name'] | default(_('[Unknown artist]')) }}
+    <dd>     
+      <a href="{{ url_for('artist.entity', mbid=review.entity['id']) }}" target="_blank">{{ entity['name'] }}</a>
```

Comment:
I think this is an error here, it should be `mbid=entity['id']`, not `review.entity`.

Be careful with the line above, too - You can see in the diff view that you've added some whitespace after the `dd` tag